app/controllers/timelineable.rb

```diff
@@ -6,13 +6,13 @@ module Timelineable
   end
 
   def timeline
-    @display_date = display_date_as_time.to_date
+    @display_date = display_date_as_time
```

Comment:
I'm open to giving it a new name, but I can't think of anything good. Any ideas?

Other than the I18n formatting, where did it get you?